### ISSUE TITLE
Min sizes in tables 6.2 and 6.3

### ISSUE THREAD

**Issue by diefenbecker** [INTERPOL-Innovation-Centre/ANSI-NIST-XML-ITL-Implementation#20]
What about adding minimum sizes for the given finger and palm position in the tables 6.2 and 6.3?
This is to be more compliant to some European systems like SIS AFIS which do have minimum sizes for the images.

**Comment by tsaracouvert** [comment]
Yes from the group. To be investigate from FBI, Schengen, Individual country, provider,...
